TF_Service_dotNet/TouchFree/TrackingConnectionManager.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Leap;
+using Ultraleap.TouchFree.Library.Configuration;
+
+namespace Ultraleap.TouchFree.Library
+{
+    public class TrackingConnectionManager
+    {
+        Controller controller;
```

Comment:
Given the WebSocket troubles we ran into yesterday, it may be worth specifying this as a (Leap?) controller here to avoid any confusion in integration (I know this is internal to the Service but it **is** opensource)